apps/admin-x-activitypub/src/api/activitypub.ts

```diff
@@ -407,12 +376,12 @@ export class ActivityPubAPI {
         return this.getPaginatedPosts('.ghost/activitypub/inbox', next);
     }
 
-    async getPostsByAccount(next?: string): Promise<PaginatedPostsResponse> {
-        return this.getPaginatedPosts('.ghost/activitypub/posts', next);
+    async getPostsByAccount(handle: string, next?: string): Promise<PaginatedPostsResponse | GetProfilePostsResponse> {
```

Comment:
This endpoint returns a list of `Activity` for remote profiles and a list of `Posts` for local profiles. I went with this approach because mapping activity to `postDTO` in the backend and then converting it back to activity here just felt like extra work. 
I did try mapping it though, but it wasn’t super smooth and caused a bunch of UI issues that needed a lot of manual testing. Doing it this way made testing and shipping a lot easier.